log.go

```diff
@@ -70,19 +78,19 @@ func (l *Logger) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.Ha
 
 	res := rw.(negroni.ResponseWriter)
 
-	if res.Status() >= 400 {
-		grip.Warning(message.Fields{
-			"method":   r.Method,
-			"request":  reqID,
-			"path":     r.URL.Path,
-			"duration": time.Since(start),
-			"action":   "completed",
-			"status":   res.Status(),
-			"remote":   remote,
-			"outcome":  http.StatusText(res.Status()),
-			"span":     time.Since(start).String(),
-		})
-	}
+	code := res.Status()
+
+	grip.Log(getLevel(code), message.Fields{
+		"method":   r.Method,
+		"request":  reqID,
+		"path":     r.URL.Path,
+		"duration": time.Since(start),
+		"action":   "completed",
+		"status":   code,
+		"remote":   remote,
+		"outcome":  http.StatusText(res.Status()),
```

Comment:
tiny nit, but evergreen logging just uses that status code. I'd also find it easier to query splunk with that. Maybe change this? Or include both?

Also, reuse the code variable here.